packages/apify/src/autoscaling/snapshotter.ts

```diff
@@ -374,13 +370,12 @@ export class Snapshotter {
     /**
      * Creates a snapshot of current CPU usage
      * using the Apify platform `systemInfo` event.
-     * @param {Object} systemInfo
-     * @ignore
-     * @protected
      * @internal
      */
-    _snapshotCpuOnPlatform(systemInfo) {
+    protected _snapshotCpuOnPlatform(systemInfo: SystemInfo) {
+        // @ts-expect-error // TODO Figure out 'property does not exist'
```

Comment:
define them in the interface, we can mark them as `@internal` to not have them in the docs. or maybe better document them in the interface via jsdoc saying those are platform only properties, idk, `@internal` could be enough.